package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "link-assets": "react-native link",
```

Comment:
This also links libraries and stuff, but if this makes it easier for you then whatevs - that said react-native link is a very commonly referenced in libraries, and simple command so not sure if it really ought to be added to scripts